### PR TITLE
Add-useragent

### PR DESCRIPTION
**Pull request by rmweir**
Use lasso's configure useragent functionality in template

**Issue:**
https://github.com/rancher/rancher/issues/38775

### CODE REVIEW

File: go.mod
Comment (cmurphy) [inline go.mod:5]: lgtm just need to merge the lasso PR and update this

**Review by MbolotSuse — COMMENTED** [review]
LGTM after the lasso fork stuff is removed.

File: go.mod
Comment (mattfarina) [reply go.mod:5]: lgtm. will approve once the lasso PR is merged

**Review by cmurphy — APPROVED** [review]

**Pull request merged**
